src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -682,8 +682,8 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
           className="align-left"
           placeholder={`Number of grid columns`}
         />
-        {errors.name ? (
```

Comment:
What if we have name-related error?

Reply:
The name related error is already added at the top of this file next to the venue name, it was copy pasted here and it wasn't showing the correct error for the columns.